cypress/cypress/e2e/submariner_uninstall.cy.js

```diff
@@ -0,0 +1,38 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
+ * Copyright (c) 2021 Red Hat, Inc.
+ ****************************************************************************** */
+
+/// <reference types="cypress" />
+
+import { submarinerClusterSetMethods } from '../views/submariner/actions/submariner_actions'
+
+describe('submariner - uninstall validation', {
+    tags: ['@submariner', '@e2e'],
+    retries: {
+        runMode: 0,
+        openMode: 0,
+    }
+}, function () {
+    before(function () {
+        cy.setAPIToken()
+        cy.clearOCMCookies()
+        cy.login()
+        
+    })
+
+    it('verify that a cluster set with deployment of submariner is exists.', { tags: ['verify'] }, function () {
+        let clusterSetName = Cypress.env('CLUSTERSET_NAME')
+        submarinerClusterSetMethods.submarinerClusterSetShouldExist(clusterSetName)
+    })
+
+    it('test the Uninstall Submariner add-ons', { tags: ['uninstall'] }, function () {
+        cy.get('.pf-c-table__check > label > input').click({multiple: true}).should('be.checked')
+        cy.get('#toggle-id').click()
+        cy.get('.pf-c-dropdown__menu-item').should('be.visible').click()
+        cy.get('.pf-c-form__actions > .pf-m-primary').click()
+
+        cy.get('[data-label="Cluster"]', {timeout: 180000}).should('not.exist')
```

Comment:
I see the maximum timeout here is 180000.
Does it checks the required state from time to time or is it wait for the full timeout?

Reply:
As long as 180000 milsec have not passed, once the clusters are deleted the test will end and succeed regardless of the timeout. 
if 180000 milsec have passed and the clusters are still visible, the test will fail.

Reply:
Got it.
Thanks.